polkadot/keystore/src/lib.rs

```diff
@@ -145,8 +149,22 @@ impl Store {
 		Ok(pair)
 	}
 
+	/// Create a new key from seed. Do not place it into the store.
+	pub fn generate_from_seed(&mut self, seed: &str) -> Result<Pair> {
```

Comment:
this should document that only the first 32 bytes are used or maybe just use the hash of the string.

Reply:
This is meant to be used for testing only. Added a comment.